app/components/OceanInterface/OceanInterface.tsx

```diff
@@ -1,19 +1,23 @@
 import { CTransactionSegWit } from '@defichain/jellyfish-transaction/dist'
 import { WhaleApiClient } from '@defichain/whale-api-client'
+import { Transaction } from '@defichain/whale-api-client/dist/api/transactions'
 import { MaterialIcons } from '@expo/vector-icons'
 import React, { useCallback, useEffect, useRef, useState } from 'react'
 import { ActivityIndicator, Animated, Linking, TouchableOpacity, View } from 'react-native'
 import { useDispatch, useSelector } from 'react-redux'
 import { Text } from '..'
-import { Logging } from '../../api/logging'
+import { Logging } from '../../api'
 import { useWallet } from '../../contexts/WalletContext'
 import { useWhaleApiClient } from '../../contexts/WhaleContext'
+import { fetchTokens } from '../../hooks/wallet/TokensAPI'
 import { RootState } from '../../store'
 import { firstTransactionSelector, ocean, OceanTransaction } from '../../store/ocean'
 import { tailwind } from '../../tailwind'
 import { translate } from '../../translations'
 
 const MAX_AUTO_RETRY = 1
+const MAX_TIMEOUT = 300000
+const TIMEOUT_INTERVAL = 30000
```

Comment:
I think we need extra config

- TIMEOUT: give up
- INTERVAL_1ST: getNetwork === playground ? 3000++ : 30000++
- INTERVAL_REST: 1000 (block should be ready anytime soon)

Reply:
Updated this @ivan-zynesis, what I did is to call an initial timeout. Then the succeeding interval, I put 5 secs so it won't spam API.